Physics2D.Diagnostics/Diagnostics/DebugView.cs

```diff
@@ -74,10 +74,13 @@ public class DebugView : DebugViewBase, IDisposable
         public Rectangle PerformancePanelBounds = new Rectangle(330, 100, 200, 100);
         private Vector2[] _background = new Vector2[4];
         public bool Enabled = true;
-        
+
         public const int CircleSegments = 32;
         private Complex circleSegmentRotation = Complex.FromAngle((float)(Math.PI * 2.0 / CircleSegments));
 
+
+        private Category _debuggableCategories;
```

Comment:
I think we should model this after the Flags property. Making it public so that the user can query the current state and/or directly set the categories.
public Category CategoryFlags { get; set; }